### PR TITLE
[3D] line3dsymbol and polygon3dsymbol: Rename height property to offset

### PR DESCRIPTION
**Pull request by ptitjano**
## Description

The `height` property is in fact a vertical offset applied to the line or the polygon. Also, the `offset` phrase is already used by the terrain for similar purposes.

cc @benoitdm-oslandia @soaubier

### CODE REVIEW

File: python/3d/auto_generated/symbols/qgsline3dsymbol.sip.in [hunk 1]
```diff
@@ -80,13 +80,13 @@ Returns width of the line symbol (in map units)
 Sets width of the line symbol (in map units)
 %End
 
-    float height() const;
+    float offset() const;
```
Comment (nyalldawson) [inline python/3d/auto_generated/symbols/qgsline3dsymbol.sip.in:83]: These are API breaks -- you'll need to keep the old ones around for compatibility and mark them as deprecated.

Comment (ptitjano) [reply python/3d/auto_generated/symbols/qgsline3dsymbol.sip.in:83]: `QgsLine3DSymbol` and `QgsPolygon3DSymbol` are considered as unstable API as defined in the header files: "warning This is not considered stable API, and may change in future QGIS releases. It is exposed to the Python bindings as a tech preview only.". 

Do we really want to keep the compatibility?

Comment (nyalldawson) [reply python/3d/auto_generated/symbols/qgsline3dsymbol.sip.in:83]: I'd still lean toward compatibility/no breakage in cases where it's trivial to do so.

**Review by benoitdm-oslandia — APPROVED** [review]
LGTM

**Comment by ptitjano** [comment]
@nyalldawson Can you take an other look at it please?

**Review by nyalldawson — APPROVED** [review]

**Pull request merged**